osu.ElasticIndexer/User.cs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using Dapper;
+using Dapper.Contrib.Extensions;
+using MySqlConnector;
+
+namespace osu.ElasticIndexer
+{
+    /// <summary>
+    /// User mapping model that contains only the properties needed for the indexer.
+    /// </summary>
+    [SuppressMessage("ReSharper", "InconsistentNaming")]
+    [SuppressMessage("Style", "IDE1006")]
+    [Table("phpbb_users")]
+    public class User : Model
+    {
+        [Computed]
+        public override long CursorValue => user_id;
+
+        [Computed]
+        public bool ShouldIndex => true;
+
+        [Computed]
+        public string country_acronym { get; set; } = string.Empty;
+
+        public string country_code => country_acronym;
+
+        public uint user_id { get; set; }
+
+        public override string ToString() => $"user_id: {user_id} country_code: {country_code}";
+
+        public static Dictionary<uint, User> FetchUserMappings(IEnumerable<SoloScore> scores)
+        {
+            var userIds = scores.Select(s => s.user_id);
+            using (var dbConnection = new MySqlConnection(AppSettings.ConnectionString))
```

Comment:
I think you can use `Database.GetConnection` here (comes from `OsuQueueProcessor` package).

Reply:
queue processor didn't have an option for password or extra connection string settings (but mostly password)

Reply:
Hmmm, I can look to add that so we have consistency in how this can be deployed in production (we never use passwords there).